### ISSUE TITLE
Suggestions on improving accuracy

### ISSUE THREAD

**Issue by qian39** [SUPER-TADORY/1p-frac#3]
Hi,

I've been fine-tuning the pre-trained model vit_tiny_patch16_224_sigma3.5_delta0.1_sample1000_80000ep on CIFAR-100, but I'm only achieving around 75% accuracy instead of the expected 84.2%. I'm using a batch size of 768 and a GPU with the following setup: NVIDIA RTX A6000, Driver Version: 535.183.01, CUDA Version: 12.2.

Do you have any suggestions on how to improve the accuracy?

Thank you!
